umbraco-cloud/set-up/project-settings/README.md

```diff
@@ -120,6 +122,21 @@ On your Umbraco Cloud project, it is possible to see the usage of Custom Domains
 
 You can see metrics related to the overall health and performance of the Azure app service hosting the live environment of your solution.
 
+## Automatic Upgrades
+
+We handle minor and patch upgrades for the Umbraco components used by Umbraco Cloud, so you don't have to. 
+Here you can control if you want to opt-in or out of automatic minor upgrades. 
+
+New projects are opt-in by default.
+
+## Backups
+
+You can handle backups from this setting. You can create a database backup of one or more of your cloud environments. 
+
+(Coming soon) It is also possible to upload your own '.bacpac' file.
```

Comment:
Maybe let's wait to add this to the page until the feature has been released on Umbraco Cloud 😊

The same is also the case with the sentence below.